runtime-modules/content/src/lib.rs

```diff
@@ -455,6 +474,8 @@ decl_module! {
             // == MUTATION SAFE ==
             //
 
+            let _ = Balances::<T>::slash(&sender, channel_state_bloat_bond);
+
```

Comment:
Do we not credit the bloat bond to some treasury account? I don't actually remember, what do we do for bounties or in the storage pallet?

Reply:
Yes, we credited to StorageTreasury, in bounties we had a Council Budget, that's why I got confused about using content workgroup budget account instead of simply the sender's account. Since, for example, channel transfer uses the budget or the sender based on which type is the owner.
I implemented with an imbalance, and always used the sender's account.
In my opinion we should use the following
Lead -> Senders account
Memeber - > Senders account
Curator Group -> Content working group budget
We should also break the functions that transfer funds in order these functions not fail, like we did in bounties